src/components/textbox/textbox.stories.mdx

```diff
@@ -229,11 +232,30 @@ The following keys are available to override the translations for this component
 <TranslationKeysTable
   translationData={[
     {
-      name: "locale",
+      name: "characterCount.tooManyCharacters",
+      description:
+        "The message displayed below the input which will inform"
+        +" users if they have excedded the set `characterLimit` and how much by." 
+        +" This will also be announced to screenreaders.",
+      type: "func",
+      returnType: "string",
+    },
+        {
+      name: "characterCount.charactersLeft",
+      description:
+        "The message displayed below the input which will inform"
+        +" users how many characters they have before they reach  or exceed the"
+        +" set `characterLimit`. This will also be announced to screenreaders.",      
+        type: "func",
+      returnType: "string",
+    },
+        {
+      name: "characterCount.visuallyHiddenHint",
       description:
-        "The locale code to display the correct number format in the counter",
+        "The message which will be read out to screen reader users, informing them"
+        +" of the set `characterLimit`.",
       type: "func",
       returnType: "string",
     },
   ]}
-/>
+/>
```

Comment:
**suggestion**

```suggestion
      name: "characterCount.tooManyCharacters",
      description:
        "The message displayed below the input which will inform"
        +" users if they have exceeded the set `characterLimit` and how much by." 
        +" This will also be announced to screen readers.",
      type: "func",
      returnType: "string",
    },
        {
      name: "characterCount.charactersLeft",
      description:
        "The message displayed below the input which will inform"
        +" users how many characters they have before they reach  or exceed the"
        +" set `characterLimit`. This will also be announced to screen readers.",      
        type: "func",
      returnType: "string",
    },
        {
      name: "characterCount.visuallyHiddenHint",
      description:
        "The message which will be read out to screen reader users, informing them"
        +" of the set `characterLimit`.",
      type: "func",
      returnType: "string",
    },
  ]}
/>
```